code/iaas/agent-instance/src/main/resources/META-INF/cattle/system-services/agent-instance-defaults.properties

```diff
@@ -50,3 +51,11 @@ serviceconsumemap.create.agentInstanceProvider.dnsService.apply=hosts
 serviceconsumemap.create.agentInstanceProvider.dnsService.increment=hosts
 serviceexposemap.create.agentInstanceProvider.dnsService.apply=hosts
 serviceexposemap.create.agentInstanceProvider.dnsService.increment=hosts
+
+nic.activate.agentInstanceProvider.dhcpService.apply=healthcheck
```

Comment:
wanted to avoid adding the extra services as I remember you were saying that you want to refactor network service code at some point...let me know what you think, and I can add an extra service if needed.